.travis.yml

```diff
@@ -49,6 +49,21 @@ jobs:
       script: travis_wait docker-compose run -e FORM_FACTOR=huge test-ci
     - name: enormous
       script: travis_wait docker-compose run -e FORM_FACTOR=enormous test-ci
+    # - stage: compile - lowlight theme
```

Comment:
can we add a comment about why and when we turn these back on :)

Reply:
Added here : https://github.com/cerner/terra-clinical/pull/608/commits/0ac5e4c81256892b50c3dead81fc4fa34daae34c